app/presenters/print/detainee_presenter.rb

```diff
@@ -0,0 +1,39 @@
+module Print
+  class DetaineePresenter < SimpleDelegator
+    def identifier
+      "#{prison_number}: #{surname}"
+    end
+
+    def cro_number
+      model.cro_number.present? ? model.cro_number : 'None'
+    end
+
+    def pnc_number
+      model.pnc_number.present? ? model.pnc_number : 'None'
+    end
+
+    def date_of_birth
+      model.date_of_birth.strftime('%d %b %Y')
```

Comment:
Just spotted a `humanized` date format matching this. We should probably use that for consistency.